packages/components-js/tests/e2e/puppeteer/specs/modal.e2e.ts

```diff
@@ -148,14 +148,15 @@ it('should have correct transform when dismissed and opened', async () => {
   const getModalTransform = async () => getElementStyle(await getModal(), 'transform', { waitForTransition: true });
 
   const initialModalTransform = await getModalTransform();
-  expect(initialModalTransform).toBe('matrix(0.9, 0, 0, 0.9, 0, 0)');
+  expect(initialModalTransform).toBe('matrix(1, 0, 0, 1, 0, 33.5)');
 
   await openModal();
   const openModalTransform = await getModalTransform();
   expect(openModalTransform).toBe('matrix(1, 0, 0, 1, 0, 0)');
   expect(initialModalTransform).not.toBe(openModalTransform);
 
   await dismissModal();
+  await new Promise((resolve) => setTimeout(resolve, CSS_TRANSITION_DURATION)); // transition delay for visibility
```

Comment:
Why? I thought more styles are overridden now compared to before.

Reply:
In this case I am not sure why it can't be removed, it is the same case for flyout..